### ISSUE TITLE
Documentation is incomplete

### ISSUE THREAD

**Issue by chris3000** [ethanejohnsons/FPV-Racing-Mod-Fabric#25]
Documentation is lacking how to actually enable FPV mode. I finally was able to figure it out by watching this video: https://www.youtube.com/watch?v=n8SqQYQqCxQ

Documentation should also point out how to change Quadz settings. The settings icon on the mod isn't easy to find.

**Comment by chris3000** [comment]
I should also say that once I got it working, it was a superb FPV simulator. I hope you continue to develop it.

**Comment by The-PPWD** [comment]
Yes, this is something that needs improvements. Quadz has gone through a lot of changes and growing pains as it's the first mod we started making; as such, keeping up on documenting things kind of got swept under the rug. I'll leave this issue open as a reminder to document how things work 1.18.1+. In the future, documentation will go on our docs page (https://docs.lazurite.dev/).

I'm glad to hear that you enjoy it! Full disclosure, we are still working on it and actually have a 1.18 branch; however, I really don't recommend using it as things are still very... weird. Quadz being made up of several components (namely Rayon and Lattice) necessitates its components getting updates and fixes before it. Currently for 1.18, Lattice "works" (air quotes) and Rayon is getting some finishing touches. Stay tuned and thanks for the kind words!
